Tests/FingerprintJSTests/FingerprintJSVersionTests.swift

```diff
@@ -22,22 +22,22 @@ final class FingerprintJSVersionTests: XCTestCase {
         let versions = [FingerprintJSVersion].since(fromVersion)
 
         // then
-        XCTAssertEqual(versions, [.v2, .v3, .v4])
+        XCTAssertEqual(versions, [.v2, .v3, .v4, .v5])
     }
 
     func test_whenAll_thenReturnsAllVersions() {
         // when
         let versions = [FingerprintJSVersion].all
 
         // then
-        XCTAssertEqual(versions, [.v1, .v2, .v3, .v4])
+        XCTAssertEqual(versions, [.v1, .v2, .v3, .v4, .v5])
     }
 
     func test_whenLatest_thenReturnsVersionFour() {
```

Comment:
Incorrect method name.

```suggestion
    func test_whenLatest_thenReturnsVersionFive() {
```

Reply:
Good catch, fixed.